daffodil-core/src/main/scala/org/apache/daffodil/dsom/SimpleTypes.scala

```diff
@@ -568,7 +569,11 @@ final class EnumerationDef(
   override lazy val optRepType = parentType.optRepType
 
   lazy val enumValueRaw: String = (xml \ "@value").head.text
-  lazy val enumValueCooked: DataValuePrimitive = parentType.primType.fromXMLString(enumValueRaw)
+  lazy val enumValueCooked: DataValuePrimitive = try {
+    parentType.primType.fromXMLString(enumValueRaw)
+  } catch {
+    case e: InvalidPrimitiveDataException => SDE("Invalid data for enumeration: %s".format(e.getMessage))
```

Comment:
Remove call to .format. Unnecessary as SDE takes a format string and varargs for the arguments to that format string.